Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+    outputs:
+    - contextPath: PrismaSase.Tag.id
+      description: The tag id.
+      type: String
+    - contextPath: PrismaSase.Tag.name
+      description: The tag name.
+      type: String
+    - contextPath: PrismaSase.Tag.folder
+      description: The tag folder.
+      type: String
+    - contextPath: PrismaSase.Tag.color
+      description: The tag color.
+      type: String
+    - contextPath: PrismaSase.Tag.comments
+      description: The tag comments.
+      type: String
+  - arguments:
+    - default: false
+      description: The specific tag to delete
+      isArray: false
+      name: tag_id
+      required: true
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    description: Delete a specific tag.
+    execution: false
+    name: prisma-sase-tag-delete
+  - arguments:
+    - default: false
+      description: A specific address group to return. If not specified, all address groups will be returned.
+      isArray: false
+      name: group_id
+      required: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
+      isArray: false
+      name: folder
+      predefined:
+      - Shared
+      - Mobile Users
+      - Remote Networks
+      - Service Connections
+      - Mobile Users Container
+      - Mobile Users Explicit Proxy
+      required: false
+    - default: false
+      isArray: false
+      name: limit
+      description: The maximum number of results to return. Default is 50.
+    - default: false
+      isArray: false
+      name: page
+      description: Page number you would like to view. Each page contains page_size values. Must be used along with page_size.
+    - default: false
+      isArray: false
+      name: page_size
+      description: Number of results per page to display.
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    execution: false
+    description: Lists all address groups.
+    name: prisma-sase-address-group-list
+    outputs:
+    - contextPath: PrismaSase.AddressGroup.id
+      description: The address group id.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.name
+      description: The address group name.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.description
+      description: The address group description.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.addresses
+      description: The address group addresses.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.dynamic_filter
+      description: The address group filter.
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASE folder location.
```
Is this the folder location or folder type/name?